client/settings/payment-settings/general-settings-section.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { React, useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardFooter,
+	CheckboxControl,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import CardBody from '../card-body';
+
+const GeneralSettingsSection = () => {
+	const [ enableStripe, setEnableStripe ] = useState( false );
+	const [ enableTestMode, setEnableTestMode ] = useState( false );
+
+	const toggleEnableStripe = () => setEnableStripe( ! enableStripe );
+	const toggleEnableTestMode = () => setEnableTestMode( ! enableTestMode );
+
+	const CheckboxDescription = styled.span`
```

Comment:
This works fine. Just noticed that in other files the `styled` elements are declared on top and out of the component itself.

Reply:
Thanks for pointing it out. I kept it inside the `GeneralSettingsSection` component since the `CheckboxDescription` is only used within the component.